src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/PublicationCacheViewModel.cs

```diff
@@ -25,4 +27,4 @@ public record PublicationCacheViewModel
     public ContactViewModel Contact { get; init; } = null!;
 
     public ExternalMethodologyViewModel? ExternalMethodology { get; init; }
-}
+}
```

Comment:
It is a unix convention that text files should end with a new line. In theory, software dealing with text files can expect text files to end with a new line. In practice, I don't think much software will make that assumption because it's broken so regularly.

Although, it's not always followed, we agreed at some point to change our editors to leave a newline at the end of each file. To be honest, it probably doesn't matter.